clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -26,7 +26,7 @@
 import java.util.Map;
 
 /**
- * Some configurations shared by both producer and consumer
+ * Configurations shared by Kafka client applications: producer, consumer and connect, etc.
```

Comment:
nit: since we are changing this, probably should be one of the following: 
`Configurations shared by Kafka client applications: producer, consumer, connect, etc.`
or 
`Configurations shared by Kafka client applications: producer, consumer, connect and adminclient.`

adminclient is one that is missing, so the version with `etc` might be preferable.

Reply:
Sounds good!